how-to-docs/drone-how-to.md

```diff
@@ -138,7 +139,9 @@ click the `+ Create New Repositories` that is:
 
 Add your project to the UKHomeOffice Quay account
 
-[Raise a ticket for a new UKHomeOffice Quay robot in the Platform Access column](https://github.com/UKHomeOffice/application-container-platform-bau/projects/1). You have to pick a name for it.  You should be supplied a robot password in response.
+[Raise a ticket for a new UKHomeOffice Quay robot on the BAU board](https://github.com/UKHomeOffice/application-container-platform-bau/issues/). You have to pick a name for it and put it along with the name of the repo you created (in Quay) in the issue. You should be supplied a robot password in response.
```

Comment:
Shouldn't this point to the relevant Support Request form on the hub?

Reply:
Oh yeah, I completely forgot about that

Reply:
👍  😸